components/browser/engine-gecko-nightly/src/main/java/mozilla/components/browser/engine/gecko/GeckoEngineSession.kt

```diff
@@ -680,7 +680,13 @@ class GeckoEngineSession(
             session: GeckoSession,
             historyList: GeckoSession.HistoryDelegate.HistoryList
         ) {
-            val items = historyList.map { HistoryItem(title = it.title, uri = it.uri) }
+            val items = historyList.map {
+                val title: String? = it.title
```

Comment:
Could you leave a comment here why we are doing this and link to the GV bug that fixes the issue.